webpack/assets/javascripts/react_app/components/HostsIndex/index.js

```diff
@@ -92,31 +124,142 @@ const HostsIndex = () => {
   RowSelectTd.propTypes = {
     rowData: PropTypes.object.isRequired,
   };
+  const dispatch = useDispatch();
+  const { destroyVmOnHostDelete } = useForemanSettings();
+  const deleteHostHandler = ({ hostName, computeId }) =>
+    dispatch(deleteHost(hostName, computeId, destroyVmOnHostDelete));
```

Comment:
This per row delete operation seems to return me to `/hosts/` while bulk delete returns me to `/new/hosts` after deleting. Do you think this should be fixed in https://github.com/theforeman/foreman/pull/9879

Reply:
Also select All does not  work for `Schedule Jobs` since the number of hosts is 0.  We'd have to address that. I am ok with pushing that for a different issue.

Reply:
Will look into it

Reply:
So I reused the single delete action from host details, which actually appears to have the same (wrong) behavior. It's here: https://github.com/theforeman/foreman/blob/38aa4bbf853ba3f6f0f57837d996be051a125cd6/webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/actions.js#L48

I think https://github.com/theforeman/foreman/pull/9879 is indeed the best place to fix it. In addition to the Ruby changes there, should do an audit of wherever we have `foremanUrl('/hosts')` (I only found 4). In those places we can `useForemanSettings()` to get the setting and set the url accordingly.